samples/TerraFX/Graphics/HelloModelViewProjection.cs

```diff
@@ -0,0 +1,277 @@
+        private readonly List<Mesh> _meshes = null!;
+        private readonly List<AppearanceStyle> _styles = null!;
+        private Appearance<Model<Box>> _box;
+        private Appearance<Model<Tetrahedron>> _tetrahedron;
+        private Camera _camera;
+        private float _meshRotationAngle;
+
+        public HelloModelViewProjection(string name, params Assembly[] compositionAssemblies)
+            : base(name, compositionAssemblies)
+        {
+            var boxTransform = Transform
+                .Identity
+                .WithScale(new Vector3(0.5f, 0.5f, 0.5f))
+                .WithTranslation(new Vector3(0.5f, 0, 0.5f));
+            boxTransform = Transform.AddRotationAroundTargetX(boxTransform, 0.1f);
+
+            var tetrahedronTransform = Transform
+                .Identity
+                .WithScale(new Vector3(0.6f, 0.6f, 0.6f))
+                .WithTranslation(new Vector3(-0.5f, 0, 0.5f));
+            tetrahedronTransform = Transform.AddRotationAroundTargetX(tetrahedronTransform, 0.1f);
+            tetrahedronTransform = Transform.AddRotationAroundTargetY(tetrahedronTransform, 0.1f);
+
+            _box = new Appearance<Model<Box>>(new Model<Box>(new Box(Vector3.One), boxTransform), AppearanceStyle.Green);
+            _tetrahedron = new Appearance<Model<Tetrahedron>>(new Model<Tetrahedron>(new Tetrahedron(Vector3.One), tetrahedronTransform), AppearanceStyle.Red);
+            _meshPrimitives = new List<GraphicsPrimitive>();
+            _meshes = new List<Mesh>();
+            _styles = new List<AppearanceStyle>();
+        }
+
+        // ----- methods.public -----
+
+        public override void Initialize(Application application, TimeSpan timeout)
+        {
+            base.Initialize(application, timeout);
+            _camera = new Camera(Transform.Identity, Window.ClientSize, new Vector2(0, 1), ProjectionMode.Orthogonal);
+
+            var graphicsDevice = GraphicsDevice;
+            var currentGraphicsContext = graphicsDevice.CurrentContext;
+
+            _meshes.Add(_box.Model.Shape.ToMesh());
+            _meshes.Add(_tetrahedron.Model.Shape.ToMesh());
+            _styles.Add(_box.Style);
+            _styles.Add(_tetrahedron.Style);
+
+            currentGraphicsContext.BeginFrame();
+            ulong vertexBufferSize = 0;
+            ulong indexBufferSize = 0;
+
+            for (var i = 0; i < _meshes.Count; ++i)
+            {
+                var mesh = _meshes[i];
+                vertexBufferSize += (ulong)(SizeOf<PosNormColorVertex>() * mesh.Vs.Length);
+                indexBufferSize += (ulong)(SizeOf<uint>() * 3 * mesh.Ts.Length);
+            }
+
+            using var vertexStagingBuffer = graphicsDevice.MemoryAllocator.CreateBuffer(GraphicsBufferKind.Default, GraphicsResourceCpuAccess.CpuToGpu, vertexBufferSize);
+            using var indexStagingBuffer = graphicsDevice.MemoryAllocator.CreateBuffer(GraphicsBufferKind.Default, GraphicsResourceCpuAccess.CpuToGpu, indexBufferSize);
```

Comment:
Newline to help with separating the using context from the field assignments.
```suggestion
            using var indexStagingBuffer = graphicsDevice.MemoryAllocator.CreateBuffer(GraphicsBufferKind.Default, GraphicsResourceCpuAccess.CpuToGpu, indexBufferSize);

```

Reply:
fixed